.changeset/moody-days-drive.md

```diff
@@ -0,0 +1,8 @@
+---
+"@medusajs/modules-sdk": minor
+"@medusajs/types": minor
+"@medusajs/utils": minor
```

Comment:
@olivermrbl Since these 3 are already at a 1.x version, based on semantic versioning they should be a minor. Should we follow a different logic instead?